api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -521,31 +527,34 @@ def touch_tip(
     ) -> InstrumentContext:
         """
         Touch the pipette tip to the sides of a well, with the intent of
-        removing left-over droplets
+        removing leftover droplets. See also :ref:`touch-tip`.
 
-        :param location: If no location is passed, pipette will
-                         touch tip at current well's edges
+        :param location: If no location is passed, the pipette will
+                         touch tip at current well's edges.
         :type location: :py:class:`.Well` or None
         :param radius: Describes the proportion of the target well's
-                       radius. When `radius=1.0`, the pipette tip will move to
-                       the edge of the target well; when `radius=0.5`, it will
+                       radius. When ``radius=1.0``, the pipette tip will move to
+                       the edge of the target well. When ``radius=0.5``, it will
                        move to 50% of the well's radius. Default: 1.0 (100%)
         :type radius: float
-        :param v_offset: The offset in mm from the top of the well to touch tip
-                         A positive offset moves the tip higher above the well,
-                         while a negative offset moves it lower into the well
-                         Default: -1.0 mm
+        :param v_offset: How far above or below the well to touch the tip, measured in mm.
+                         A positive offset moves the tip higher above the well.
+                         A negative offset moves the tip lower into the well.
+                         Default is -1.0 mm.
         :type v_offset: float
         :param speed: The speed for touch tip motion, in mm/s.
-                      Default: 60.0 mm/s, Max: 80.0 mm/s, Min: 20.0 mm/s
+
+                        - Default: 60.0 mm/s
+                        - Max: 80.0 mm/s
+                        - Min: 1.0 mm/s
```

Comment:
I like spelling these out.
```suggestion
                        - Maximum: 80.0 mm/s
                        - Minimum: 1.0 mm/s
```